packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -501,6 +486,92 @@ async function handleGitPreference(gitInitFlag) {
   }
 }
 
+async function doesDirectoryAlreadyExist(appDir, overwrite) {
+  let newAppDir = appDir
+
+  // Check if the new app directory already exists
+  if (fs.existsSync(newAppDir) && !overwrite) {
+    // Check if the directory contains files and show an error if it does
+    if (fs.readdirSync(newAppDir).length > 0) {
+      tui.stopReactive(true)
+      tui.displayWarning(
+        'Project directory already contains files',
+        [
+          `'${RedwoodStyling.info(newAppDir)}' already exists and is not empty`,
+        ].join('\n')
+      )
+
+      try {
+        const response = await tui.prompt({
+          type: 'select',
+          name: 'projectDirectoryAlreadyExists',
+          message: 'How would you like to proceed?',
+          choices: [
+            'Quit install',
+            'Overwrite files and continue install',
+            'Specify a different directory',
+          ],
+          initial: 0,
+        })
+
+        // overwrite the existing files
+        if (
+          response.projectDirectoryAlreadyExists ===
+          'Overwrite files and continue install'
+        ) {
+          // blow away the existing directory and create a new one
+          await fs.remove(newAppDir)
+        } // specify a different directory
+        else if (
+          response.projectDirectoryAlreadyExists ===
+          'Specify a different directory'
+        ) {
+          const newDirectoryName = await handleNewDirectoryNamePreference()
+          newAppDir = path.resolve(process.cwd(), newDirectoryName)
```

Comment:
Sorry for being lazy and not actually testing this myself, but does this support absolute dirs? Like if the user specifies `/Users/tobbe/tmp/test-project`

Reply:
I stopped being lazy and tried it myself 😁 
It does work with absolute paths too ✅